### PR TITLE
🏗 Add opt-in option to nightly release channel

### PR DESCRIPTION
**Pull request by danielrozenberg**
Related to #25616

### CODE REVIEW

File: tools/experiments/experiments.js [hunk 1]
```diff
@@ -290,6 +293,8 @@ function isExperimentOn_(id) {
       return [AMP_OPT_IN_COOKIE.BETA, _LEGACY_AMP_OPT_IN_COOKIE.BETA].includes(
```
Comment (rsimha) [inline tools/experiments/experiments.js:293]: Curious: Can the legacy cookie stuff go away?

File: tools/experiments/experiments.js [hunk 2]
```diff
@@ -388,8 +401,9 @@ function showConfirmation_(message, callback) {
 
 /**
  * Loads the AMP_CONFIG objects from whatever the v0.js is that the user has
- * (depends on whether they opted into beta, experimental, or a specific RTV) so
- * that experiment state can reflect the default activated experiments.
+ * (depends on whether they opted into one of the pre-release channels or a into
```
Comment (rsimha) [inline tools/experiments/experiments.js:404]: nit: "... channels or into" (remove "a")

File: tools/experiments/experiments.js (hunk 1 above)
Comment (danielrozenberg) [reply tools/experiments/experiments.js]: Yep, done!

File: tools/experiments/experiments.js (hunk 2 above)
Comment (danielrozenberg) [reply tools/experiments/experiments.js]: done

**Pull request merged**